fragments/tap-workload/accelerator.axl

```diff
@@ -0,0 +1,12 @@
+engine {
+  Exclude({"README.md"})
+  if(#bsGitRepository > "") {
```

Comment:
```suggestion
  if(#bsGitRepository != "") {
```

Reply:
So, for this I think we should do one of these:

```
        ReplaceText(regex: {pattern: "(?m)url: .*$", with: 'url: ' + #repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]})
        ReplaceText(regex: {pattern: "(?m)branch: .*$", with: 'branch: ' + #bsGitBranch})
```

or 

```
        OpenRewriteRecipe(recipe: "org.openrewrite.yaml.ChangeValue", options: {oldKeyPath: "$.spec.source.git.url", value: #repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]})
        OpenRewriteRecipe(recipe: "org.openrewrite.yaml.ChangeValue", options: {oldKeyPath: "$.spec.source.git.ref.branch", value: #bsGitBranch})
```

The OpenRewrite one takes significantly longer to process

Reply:
Using `if(#bsGitRepository != "")` gives this error from the engine

```
"org.springframework.expression.spel.SpelEvaluationException: EL1012E: Cannot index into a null value"
```

Reply:
Well, indeed the original YAML version compared to `null`, not empty string